spec/controllers/items_controller_spec.rb

```diff
@@ -89,9 +89,29 @@
           do_request
         end
 
-        it "gives the user the option to submit a request for approval" do
-          expect(assigns[:add_to_cart]).to be_blank
-          assert_redirected_to(new_facility_product_training_request_path(facility, item))
+        around(:each) do |example|
```

Comment:
Is there a way we can come up with a better technique than duplicating this everywhere? We have a `include_context "feature enabled", :create_users`, in one place, but I'm not sure that's the best option. Is there something we could do with metadata?

```
context "with training requests", feature_setting: { training_requests: true } do
```
or something similar; that was just the first thing that popped in my mind.